index.html

```diff
@@ -0,0 +1,146 @@
+        <li><a href="#program " onClick="menuClose(1);">Program</a></li>
+        <li><a href="#speakers " onClick="menuClose(1);">Heros</a></li>
+        <li><a href="#sponsors" onClick="menuClose(1);">Sponsor</a></li>
+        </ul>
+
+        </div>
+      
+
+    <main>
+        <section id="cover" class="cover">
+        <h2><img src="./images/main_big.png" alt="title" /> </h2>
+
+            <div class="cover_extra">
+                This event aims to bring together business leaders and teams who want to learn and understand how
+                teamwork is managed
+                around the world for three days of exciting talks and impressive presentations.
+            </div>
+
+
+            <p>2022.02.18(SAT)- 20(MON) </p>
+            <p>Gran Salon Corferias, Bogotá-Colombia</p>
+
+
+
+        </section>
+        <section id="program" class="program">
+
+            <div>Main Program</div>
+
+            <div class="program_out">
+                <div class="program_in">
+                    <div><img src="./images/icon_01.png" alt="title" /></div>
+                    <p>Lecture</p>
+                    <h4> We will present global trends and developments and feature articles of interest that reflect
+                        the current state of design.</h4>
+                </div>
+                <div class="program_in">
+                    <div> <img src="./images/icon_02.png" alt="title" /></div>
+                    <p>Exhibition</p>
+                    <h4>Exhibition of works in different finishes by Colombian leads who have participated in different
+                        fairs around the world.
+                    </h4>
+
+                </div>
+                <div class="program_in">
+                    <div><img src="./images/icon_03.png" alt="title" /></div>
+                    <p>Forums</p>
+                    <h4> Designers from various contries speaking about their experiences and new trends.</h4>
+                </div>
+                <div class="program_in">
+                    <div><img src="./images/icon_04.png" alt="title" /></div>
+                    <p>Worshop</p>
+                    <h4> We will do a practical workshop, where we will explore the most used techniques for design
+                        thinking.</h4>
+                </div>
+                <div class="program_in">
+                    <div><img src="./images/icon_05.png" alt="title" /></div>
+                    <p>CHPT Feedback</p>
+                    <h4> We will make a small feedback in figures of these years of activity and how the design movement
+                        has grown and we will see how it has grown.</h4>
+                </div>
+
+
+            </div>
+            <div>SEE</div>
+        </section>
+
+
+
+        <section class="speaker_heros" id="speakers">
+
+            <div class="centered-title">
+                <h2 class="black-title">Featured Heros</h2>
```

Comment:
- [ ] This section should contain a divider after the header, please consider adding one using the right color

|Your design|CC design|
|---|---|
|![Screenshot_20220520_061855](https://user-images.githubusercontent.com/25227022/169455800-2b389b24-08e0-4772-9d29-a46bdbe48567.png)|![Screenshot_20220520_061945](https://user-images.githubusercontent.com/25227022/169455868-8342aa44-d96d-481e-b2f7-41ff692cf10f.png)|